dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -76,70 +128,267 @@ def setUpClass(cls):
             ),
         )
 
-        cls.expected_props = [
```

Comment:
`expected_props` and `check_continuous_properties` have been removed now that `properties` is able to be asserted with the rest of the profile dictionary.